icu4c/source/test/intltest/intltest.cpp

```diff
@@ -1679,6 +1679,61 @@ const char *IntlTest::getSourceTestData(UErrorCode& /*err*/) {
     return srcDataDir;
 }
 
+static bool fileExists(const char* fileName) {
+    // Test for `srcDataDir` existing by checking for `srcDataDir`/message2/valid-tests.json
```

Comment:
This seems strangely, overly specific for this rather generic function.
Did this want to be a comment on or in getSharedTestData()?